src/constants/index.ts

```diff
@@ -0,0 +1,14 @@
+const BASE_URL = "https://bootcamp-api.codeit.kr/api/";
```

Comment:
해당 상수가 전역적인(constants/index.ts) 의미를 가지는 파일에 있다보니 백엔드 서버가 아니라 지금 리액트의 url을 나타낸다고 착각할수도 있을 것 같아서 요렇게 표현해주면 더 좋을 것 같네요!
```suggestion
const API_BASE_URL = "https://bootcamp-api.codeit.kr/api/";
```